LedgerSMB/Scripts/contact.pm

```diff
@@ -340,7 +341,7 @@ sub _main_screen {
                    person => $person,
                  employee => $person,
                      user => $user,
-                    roles => eval { $user->list_roles },
+                    roles => \@roles,
```

Comment:
The error that was generated was "odd number of elements in anonymous hash". No idea if the curlies were parsed as a hash or not, but I suspect they were and the curlies have only 1 element....

Thanks for reviewing and commenting on the original code.